src/test/java/test/CountTest2.java

```diff
@@ -0,0 +1,31 @@
+package test;
+
+import org.testng.*;
+import org.testng.annotations.Test;
+import org.testng.xml.XmlSuite;
+
+import java.util.List;
+
+public class CountTest2 extends SimpleBaseTest {
```

Comment:
You need to add to test to the suite.xml

Reply:
OK